### PR TITLE
Configure API host as env variables.

### PR DESCRIPTION
**Pull request by sultan-duisenbay**
Lets us use different API URLs for dev, test, prod in the future

### CODE REVIEW

File: web/web-dev-server.config.mjs [hunk 1]
```diff
@@ -18,8 +23,10 @@ export default /** @type {import('@web/dev-server').DevServerConfig} */ ({
   appIndex: 'index.html',
 
   plugins: [
-    /** Use Hot Module Replacement by uncommenting. Requires @open-wc/dev-server-hmr plugin */
-    // hmr && hmrPlugin({ exclude: ['**/*/node_modules/**/*'], presets: [presets.litElement] }),
+    replace({
+      'process.env.API_HOST': '"testgrid-data.k8s.io"',
```
Comment (chases2) [inline web/web-dev-server.config.mjs:27]: One nit: consider adding something in the README about local development that points to this file. That's how I'd run the UI against a local copy of the API, right?

**Review by chases2 — APPROVED** [review]
This looks like an excellent development improvement to me. Thanks!
/lgtm
/hold for nits

File: web/web-dev-server.config.mjs (hunk 1 above)
Comment (sultan-duisenbay) [reply web/web-dev-server.config.mjs:27]: In the future, yes. README update sounds great!

Comment (sultan-duisenbay) [reply web/web-dev-server.config.mjs:27]: I will add it as a TODO

**Pull request merged**